media/video-stitcher/test/stitcher.test.js

```diff
@@ -116,6 +121,82 @@ before(() => {
       }
     });
 
+  // Delete existing test VOD configs more than an hour old.
+  const vodConfigs = execSync(
+    `node listVodConfigs.js ${projectId} ${location}`,
+    {cwd}
+  );
+
+  vodConfigs
+    .toString()
+    .split(/\r?\n/)
+    .forEach(line => {
+      if (
+        line.includes(`locations/${location}/vodConfigs/${vodConfigIdPrefix}`)
+      ) {
+        this.nextId = line.split('/').pop();
+        let createTime = this.nextId.split('-').pop();
+        createTime = parseInt(createTime);
```

Comment:
If you don't mind, please use `Number(createTime)` instead of `parseInt`. It's not in our JavaScript style guide (no longer being updated), but the recommendation is now in our TypeScript guide (https://google.github.io/styleguide/tsguide.html#type-coercion).

Reply:
Done